config.entry.json

```diff
@@ -22,5 +22,6 @@
   "reversePrice": "0.001",
   "reverseClaimInterval": 3600,
   "reverseSubscriptionDuration": 180,
-  "reverseSubscriptionFee": "0"
+  "reverseSubscriptionFee": "0",
+  "minChargingUnit": 100
```

Comment:
This shouldn't be configurable, because mismatch between entry and exit will cause service provider to close connection

Reply:
Also the meaning of `minChargingUnit` is not clear. Suggest to change to `DefaultNanoPayUpdateThreshold`